src/modules/auth/auth.types.ts

```diff
@@ -21,6 +21,7 @@ export type AuthVerifyEmail = {
 
 export type AuthForgotPass = {
   email: string;
+  hcaptcha_token?: string;
```

Comment:
Good question. The parameter is optional because when running locally, we bypass the captcha verification. `authService.verifyHcaptcha` does the validation for us in the deployed environment, while short circuits to true in local environment